Fix:

error: implicit declaration of function 'main_loop' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- src/tfio.c.orig	2024-08-02 17:57:42.000000000 -0500
+++ src/tfio.c	2024-08-02 18:00:25.000000000 -0500
@@ -37,6 +37,7 @@
 #include "search.h"	/* queues */
 #include "tfio.h"
 #include "tfselect.h"
+#include "socket.h"
 #include "output.h"
 #include "attr.h"
 #include "macro.h"	/* macro_body() */
